tests/ui/unnested_or_patterns.rs

```diff
@@ -6,7 +6,8 @@
     clippy::cognitive_complexity,
     clippy::match_ref_pats,
     clippy::upper_case_acronyms,
-    clippy::needless_if
+    clippy::needless_if,
+    clippy::manual_range_pattern
```

Comment:
How much does this lint conflict with `unnested_or_patterns`?

Reply:
`unnested_or_patterns` specifically complains about patterns like `Some(1) | Some(2) | Some(3)` and suggests changing it to `Some(1 | 2 | 3)`, which then would cause `manual_range_pattern` to lint on the fixed code and suggest changing it to `Some(1..=3)`

Reply:
Ah, so the `allow` here is for the fixed file. Perfect. In that case, no complains. Thanks for the explanation.